### PR TITLE
See more feature

### PR DESCRIPTION
**Pull request by Hugovs2000**
This PR deals with the see more page that is navigated to from the home page when a user wants to see either more seasonal mangas or latest updates.

### CODE REVIEW

File: src/modules/SeeMorePage/components/Skeletons/SeeMoreSkeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <div className="skeleton w-20 h-32"></div>
+        <div className="skeleton w-20 h-32"></div>
+        <div className="skeleton w-20 h-32"></div>
+        <div className="skeleton w-20 h-32"></div>
+        <div className="skeleton w-20 h-32"></div>
+        <div className="skeleton w-20 h-32"></div>
+        <div className="skeleton w-20 h-32"></div>
+        <div className="skeleton w-20 h-32"></div>
+        <div className="skeleton w-20 h-32"></div>
+        <div className="skeleton w-20 h-32"></div>
```
Comment (Julianna-Venter) [inline src/modules/SeeMorePage/components/Skeletons/SeeMoreSkeleton.tsx:17]: there are ways to do this a bit better, but they're more time consuming (which is probably unnecessary here). If it adds value for you, maybe do a loop or tailwind class of some sort. Although it's not a requirement.

File: src/routeTree.gen.ts [hunk 2]
```diff
@@ -41,6 +42,11 @@ const DetailsIndexRoute = DetailsIndexImport.update({
   getParentRoute: () => rootRoute,
 } as any)
 
+const SeeMoreSectionRoute = SeeMoreSectionImport.update({
+  path: '/see-more/$section',
+  getParentRoute: () => rootRoute,
+} as any)
```
Comment (Julianna-Venter) [inline src/routeTree.gen.ts:48]: change `any`s to `unknown`s

**Review by Julianna-Venter — APPROVED** [review]
minor comments, one "any" that needs to be changed (yes it is auto generated, but should still be changed) Otherwise it looks good to me.

File: src/modules/SeeMorePage/components/Skeletons/SeeMoreSkeleton.tsx (hunk 1 above)
Comment (CharelMarais) [reply src/modules/SeeMorePage/components/Skeletons/SeeMoreSkeleton.tsx:17]: I agree, definitely do this as a loop.

File: src/modules/SeeMorePage/SeeMorePage.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  const resultingSeasonString = topData?.comicsByCurrentSeason.season
+    ? `${topData.comicsByCurrentSeason.season.charAt(0).toUpperCase() + topData.comicsByCurrentSeason.season.slice(1)}`
+    : "Seasonal";
+
+  if (loadingLatestsUpdates || loadingTopData) {
+    return <SeeMoreSkeleton />;
+  }
+
+  if (section === "latest-updates") {
+    return (
```
Comment (CharelMarais) [inline src/modules/SeeMorePage/SeeMorePage.tsx:30]: I would suggest moving these returns into their own components and returning the element component something like `<SeeMoreLatest />` `<SeeMoreSeasonal />` `<SeeMoreNotFound />` like what you did for `<SeeMoreSkeleton />`

**Review by CharelMarais — CHANGES_REQUESTED** [review]
Just a few comments, please look at:
- The any's
- The redundant code that can be a loop
- Splitting TSX returns to their own comps (Single Responsibility)

File: src/modules/SeeMorePage/SeeMorePage.tsx (hunk 3 above)
Comment (Hugovs2000) [reply src/modules/SeeMorePage/SeeMorePage.tsx:30]: I agree, will do 👍🏻

File: src/modules/SeeMorePage/components/Skeletons/SeeMoreSkeleton.tsx (hunk 1 above)
Comment (Hugovs2000) [reply src/modules/SeeMorePage/components/Skeletons/SeeMoreSkeleton.tsx:17]: Cool thing 👍🏻

File: src/routeTree.gen.ts (hunk 2 above)
Comment (Hugovs2000) [reply src/routeTree.gen.ts:48]: I'm not sure how to change this since the routeTree.gen.ts file regenerates itself each time I run the code?

**Pull request merged**